addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -78,29 +78,17 @@ function factory(dependencies) {
          * @param {function} res
          */
         async handleAddChannelAutocompleteSource(req, res) {
-            const value = req.term;
-            const escapedValue = owl.utils.escape(value);
-            this.update({ addingChannelValue: value });
```

Comment:
This line must be kept, the name of a new channel is still empty after creating it in this flux.

Reply:
I move it in "searchChannelsToOpen". It's ok ?